### PR TITLE
Adding desponda as a milestone maintainer

### PR DESCRIPTION
**Pull request by desponda**
/cc @kubernetes/release-team-leads
/cc @erismaster

### CODE REVIEW

**Comment by erismaster** [comment]
/lgtm

/approve

Welcome to the release team!

**Comment by erismaster** [comment]
https://github.com/kubernetes/org/issues/2424 is open to get access to the org - once closed the pull-org-test-all check should pass.

File: config/kubernetes/sig-release/teams.yaml
Comment (savitharaghunathan) [inline config/kubernetes/sig-release/teams.yaml:54]: nit: Change `# 1.21 RT Bug Triage Shadow` to `# 1.21  Bug Triage Shadow`

**Review by savitharaghunathan — CHANGES_REQUESTED** [review]

**Comment by palnabarun** [comment]
/hold for resolution on https://github.com/kubernetes/org/pull/2425#pullrequestreview-569548256

@erismaster -- wondering if we should onboard all the shadows as a single commit specifying the context of why they are being added as milestone-maintainers? This would help the broader reviewer and approver community to have context. Right now, with separate additions, only the RT has context.

File: config/kubernetes/sig-release/teams.yaml
Comment (palnabarun) [reply config/kubernetes/sig-release/teams.yaml:54]: @savitharaghunathan -- I think keeping `RT` is fine here since the `milestone-maintainers` list cuts across the community and is not limited to the context of the Release Team. WDYT?

**Comment by palnabarun** [comment]
/close

(ref: https://github.com/kubernetes/org/pull/2425#issuecomment-762008384)

**Pull request closed without merging**
